### DISCUSSION TITLE
Advanced Import Settings window has awkward 3D camera controls

### DISCUSSION THREAD

**Discussion by Shnazzy** [godotengine/godot-proposals#7927] [Editor]
The viewport preview of the Advanced Import Settings window doesn't feel very intuitive in terms of how mouse movement changes the camera.  Mainly, it doesn't match the main 3D viewport in the editor's rotation where by default it keeps the Y axis pointing up.

Example:

https://github.com/godotengine/godot-proposals/assets/22860318/814adf47-fa0d-48b4-84dc-3b60d94e172a

I'd like to make a pull request out of the following branch which fixes the problem, but I'd like to first make sure I wasn't looking over anything like a use case that prefers the current way it works.

Fix Branch:
https://github.com/Shnazzy/godot/tree/fix-rotation-order-advanced-import

Fix Demo:

https://github.com/godotengine/godot-proposals/assets/22860318/9ee5ec03-92e5-4245-8922-ad8872eba0d2

**Comment by mieldepoche** [comment]
yeah the current behavior doesn't make sense to me. I'd be happy with that change.

**Comment by Calinou** [comment]
For reference, a pull request was opened: https://github.com/godotengine/godot/pull/82591
